tests/app/main/test_services.py

```diff
@@ -126,53 +127,10 @@ def test_should_redirect_to_login_if_not_logged_in(self):
                      '?next=%2Fsuppliers%2F')
 
 
-class TestServicesLogin(BaseApplicationTest):
```

Comment:
This class was only checking 2 things:
1. That, once logged in, a user could _see_ an 'Edit Service Status' page
2. That, if not logged in, a user would be redirected.

Since there was a lot of overlap, I just absorbed both checks into my `TestSupplierUpdateService` class.